src/app/modules/dashboard/components/selected/selected.component.ts

```diff
@@ -37,6 +39,13 @@ export class DashboardSelectedComponent implements OnInit {
 
     ngOnInit() {
         this.slimLoadingBarService.start();
+        if (this.profileService.isAuthorized()) {
+            this.profile = this.profileService.getProfileFromLocal();
+        } else {
```

Comment:
AFAK, selected component will not work if user isn't logged yet. So, why we need an 'else' here?